addons/l10n_pe_website_sale/controllers/main.py

```diff
@@ -0,0 +1,104 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import http
+from odoo.http import request
+
+from odoo.addons.website_sale.controllers.main import WebsiteSale
+
+
+class L10nPEWebsiteSale(WebsiteSale):
+
+
+    @http.route(
+        ['/shop/peruvian_company'],
+        type="json",
+        auth="public",
+        methods=["POST"],
+        website=True
+    )
+    def company_info(self, **kw):
+        is_peruvian_company = request.website.company_id.country_id.id == request.env.ref("base.pe").id
+        return {"is_peruvian_company": is_peruvian_company}
```

Comment:
We can remove this controller because the value is passed from the template (see below)
```suggestion

```